scalameta/dialects/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -88,17 +94,31 @@ package object dialects {
     def name = "Scala211"
     def bindToSeqWildcardDesignator = Scala210.bindToSeqWildcardDesignator
     def allowXmlLiterals = Scala210.allowXmlLiterals
+    def allowInline = false
     def allowSpliceUnderscore = true // SI-7715, only fixed in 2.11.0-M5
     def allowToplevelTerms = Scala210.allowToplevelTerms
     def toplevelSeparator = Scala210.toplevelSeparator
     def metalevel = Metalevel.Zero
     private def writeReplace(): AnyRef = new Dialect.SerializationProxy(this)
   }
 
+  @leaf implicit object Paradise extends Dialect {
```

Comment:
Just a really quick comment.

This seems to depend heavily on Scala 2.11. With Scala 2.12 in RC now, 2.12 is just around the corner. Can we change the name of this to Paradise_211? Also can we write the same code to work for both Paradise_211 and Paradise_212? @xeno-by correct me if I'm confused. I may have missed something in the Pre-SIP